system-test/datastore.ts

```diff
@@ -1130,6 +1369,110 @@ describe('Datastore', () => {
     });
   });
 
+  describe('querying the datastore with an overflow data set', () => {
+    const keys = [
+      // Paths:
+      ['Rickard'],
+      ['Rickard', 'Character', 'Eddard'],
+    ].map(path => {
+      return datastore.key(['Book', 'GoT', 'Character'].concat(path));
+    });
+    const characters = [
+      {
+        name: 'Rickard',
+        family: 'Stark',
+        // eslint-disable-next-line @typescript-eslint/no-loss-of-precision
+        appearances: 9223372036854775807,
+        alive: false,
+      },
+      {
+        name: 'Eddard',
+        family: 'Stark',
+        // eslint-disable-next-line @typescript-eslint/no-loss-of-precision
+        appearances: 9223372036854775807,
+        alive: false,
+      },
+    ];
+    before(async () => {
+      const keysToSave = keys.map((key, index) => {
+        return {
+          key,
+          data: characters[index],
+        };
+      });
+      await datastore.save(keysToSave);
+    });
+    after(async () => {
+      await datastore.delete(keys);
+    });
+    it('should run a sum aggregation', async () => {
+      const q = datastore.createQuery('Character');
+      const aggregate = datastore
+        .createAggregationQuery(q)
+        .addAggregation(AggregateField.sum('appearances'));
+      const [results] = await datastore.runAggregationQuery(aggregate);
+      assert.deepStrictEqual(results, [{property_1: -18446744073709552000}]);
```

Comment:
I can confirm that this value and the value for average are the same values received by the grpc call so the values you see here are strictly backend calculations.